datalad/support/gitrepo.py

```diff
@@ -697,7 +710,7 @@ def __eq__(self, obj):
     @classmethod
     def is_valid_repo(cls, path):
         """Returns if a given path points to a git repository"""
-        return exists(opj(path, '.git', 'objects'))
+        return exists(opj(path, '.git'))
 
```

Comment:
Is this still good enough? We used `.git/objects` to avoid matching just anything that has a `.git` subdirectory. Any reason to change this?